version.php

```diff
@@ -17,7 +17,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2016052000;
+$plugin->version = 2017031500;
 $plugin->release = '2.2+ (Build: 2016052000)';
```

Comment:
Bump the build number up to match the version number (not really all that important, but ...)